### DISCUSSION TITLE
How can I ask/force the LLM to respond without whitespace in the JSON?

### DISCUSSION THREAD

**Discussion by Taytay** [567-labs/instructor#591] [Q&A]
I didn't see this in the docs. I've noticed that the LLM (Claude in this case) is responding with formatted/indented JSON. It wastes precious tokens to respond with whitespace in the JSON. When I hand-format prompts, I ask it to respond with JSON without whitespace to save on the newline and indent tokens, but in my first attempt at doing the same in Anthropic's Claude, it didn't seem to work. I'm going to iterate more and try other LLMs, but I thought I'd ask because I can't be the only one.

**Comment by jxnl** [comment]
anthropic tool uses xml and reformats it as json. its all a lie
